mne_bids_pipeline/_import_data.py

```diff
@@ -449,7 +449,6 @@ def import_er_data(
             cfg=cfg,
             bids_path_bads=bids_path_er_bads_in,
         )
-    raw_er.pick("meg", exclude=[])
```

Comment:
For some reason we were only picking MEG channels in empty room data. To me this step seems unnecessary, and also breaks artifact regression, so I think we should drop it.